server/models/handlers/school.js

```diff
@@ -0,0 +1,13 @@
+// Create stub which returns an array of Worker ID's given an input of school ID or school Name
+async function inputSchool(schoolID, schoolName) {
+    
+    var did_it_work = false;
+          //* insert (schoolID or schoolName)
+    did_it_work = true;
+          var workerID=[8000000,8000001,8000002,8000003]
```

Comment:
Hi Simran,
Did you mean to update the return statement to return the array of workerIds instead of did_it_work? Also I would recommend updating the variable naming to be workerIds to increase clarity that it's an array of workers.

Reply:
Hi Amy! Thank you for your comments! I've changed the variable name to workerIds and rather than did_it_work I have replaced it with array_of_workerIds
Is that what you were looking for?

Reply:
Hi Simran, 
Yes, but I am confused by the new update. I have made a new comment in that respective area.